test/packages/bad_deployment_mode_without_identities/manifest.yml

```
@@ -0,0 +1,49 @@
+format_version: 3.1.4
```

```suggestion
format_version: 3.2.2
```
This should fix this error:
```
Additional property default is not allowed
```